packages/storybook/stories/QuickStarts/QuickstartStatefulCall.stories.mdx

```diff
@@ -0,0 +1,173 @@
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| [CallClient](https://docs.microsoft.com/javascript/api/azure-communication-services/@azure/communication-calling/callclient?view=azure-communication-services-js) | Low-level call library client.                                                         |
+| [CallAgent](https://docs.microsoft.com/javascript/api/azure-communication-services/@azure/communication-calling/callagent?view=azure-communication-services-js)   | Low-level call agent for calling client.                                               |
+| [Call](https://docs.microsoft.com/javascript/api/azure-communication-services/@azure/communication-calling/call?view=azure-communication-services-js)             | Low-level call object for calling client                                               |
+| [createStatefulCallClient](./?path=/story/statefulclient-overview--page)                                                                                          | Method to translate low-level library client into a stateful client for the UI Library |
+| [CallClientProvider](./?path=/story/statefulclient-reacthooks-setup--page)                                                                                        | Provider allows access to the Stateful Call Client to the components inside of it      |
+| [CallAgentProvider](./?path=/story/statefulclient-reacthooks-setup--page)                                                                                         | Provider allows access to the Stateful Call Agent to the components inside of it       |
+| [CallProvider](./?path=/story/statefulclient-reacthooks-setup--page)                                                                                              | Provider allows access to the Stateful Call object to the components inside of it      |
+| [usePropsFor](./?path=/story/statefulclient-reacthooks-usepropsfor--page)                                                                                         | Hook to generate required props to enable UI Components                                |
+
+## Instantiate Stateful Call Client, Call Agent and Call
+
+Update the `App.tsx` file to create a `statefulCallClient` off the low-level `CallClient` using the method `createStatefulCallClient`.
+Make sure to populate the constants at the top of the App code with information from your Azure Communication Services Resource.
+These properties will be used to instantiate the `CallClient`, `CallAgent` and `Call`.
+
+As part of the constants that need to be populated is the `groupId` which is a GUID generated by the developer.
+
+`App.tsx`
+
+<Source code={CallAppStateful} />
+
+## Add Providers
+
+Next we will add four providers that will encapsulate our UI components.
```

Comment:
We should likely reference React Context here, since that's what the providers really are.